includes/class-wau-admin-settings.php

```diff
@@ -72,6 +72,14 @@ public function addon_settings_api_init() {
 				'wau_addon_settings_section'
 			);
 
+			add_settings_field(
+				'wau_settings_check',
+				__( 'Do not send link in email', 'woo-addon-uploads' ),
+				array( &$this, 'wau_settings_enable_renderer_2' ),

```

Comment:
Please use logical name instead of `wau_settings_enable_renderer_2`

Use a name which corresponds to the function working.